src/query/models/config.go

```diff
@@ -38,7 +37,7 @@ func (t IDSchemeType) Validate() error {
 		return errors.New("id scheme type not set")
 	}
 
-	if t >= TypeLegacy && t <= TypeGraphite {
+	if t >= TypeQuoted && t <= TypeGraphite {
```

Comment:
nit: unfortunately I think we need to keep the legacy scheme, just not have it be default or required